src/plot_api/subroutines.js

```diff
@@ -272,6 +272,7 @@ exports.lsInner = function(gd) {
          *    -----
          *     x2
          */
+        var xPath = 'M0,0';
```

Comment:
maybe a good candidate for a new addition to `src/constants/` (e.g. `blankPath`)?

Reply:
We do use it a fair amount... though we use it for purposes other than just blank (as the start of a real path for example), and between the open `'M0,0'` and closed `M0,0Z` variants (though maybe we don't need the closed one and could switch to open?), I feel like hiding it in a constant may make its purpose less clear rather than more.